sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/models/EmbeddingEncodingFormat.java

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
+package com.azure.ai.openai.models;
+
+import com.azure.core.annotation.Generated;
+import com.azure.core.util.ExpandableStringEnum;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import java.util.Collection;
+
+/**
+ * Represents the available formats for embeddings data on responses.
+ */
+public final class EmbeddingEncodingFormat extends ExpandableStringEnum<EmbeddingEncodingFormat> {
```

Comment:
This should be in implementation package..

Reply:
offline discussion: made it a package-private class